packages/compass-e2e-tests/helpers/insert-data.ts

```diff
@@ -1,168 +1,217 @@
 import { MongoClient } from 'mongodb';
 import type { Db, MongoServerError } from 'mongodb';
 
-const CONNECTION_URI = 'mongodb://127.0.0.1:27091';
-
-export async function dropDatabase(db: Db | string) {
-  const database = typeof db === 'string' ? client.db(db) : db;
-  try {
-    await database.dropDatabase();
-  } catch (err) {
-    const codeName = (err as MongoServerError).codeName;
-    if (codeName !== 'NamespaceNotFound') {
-      throw err;
-    }
-  }
+import {
```

Comment:
Fair amount of cleanup, refactoring and changes in this file. Probably easiest to review just the file as if it is new.